deploy/crds/jaegertracing_v1_jaeger_crd.yaml

```diff
@@ -11,3 +11,7 @@ spec:
     singular: jaeger
   scope: Namespaced
   version: v1
+  versions:
```

Comment:
Any implications with this, if updated in environment that already has the crd deployed?

Reply:
To be honest, I didn't pay much attention to it before your comment, as t his new node is created by the CRD generator (used by the openapi generator, AFAIK). 

Looking at [the docs](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning/), I understand this has the same effect as before, as our CRD has only one version. In the future, we might have a CRD for `v2` that supports `v1` as well (served).